source/common/tracing/http_tracer_impl.cc

```diff
@@ -126,11 +124,13 @@ void HttpConnManFinalizerImpl::finalize(Span& span) {
   // Pre response data.
   if (request_headers_) {
     span.setTag("guid:x-request-id", std::string(request_headers_->RequestId()->value().c_str()));
-    span.setTag("request_line", buildRequestLine(*request_headers_, request_info_));
-    span.setTag("host_header", valueOrDefault(request_headers_->Host(), "-"));
+    span.setTag("http.url", buildUrl(*request_headers_));
+    span.setTag("http.method", request_headers_->Method()->value().c_str());
     span.setTag("downstream_cluster",
                 valueOrDefault(request_headers_->EnvoyDownstreamServiceCluster(), "-"));
     span.setTag("user_agent", valueOrDefault(request_headers_->UserAgent(), "-"));
+    span.setTag("protocol",
```

Comment:
Was thinking about keeping it more generic as not sure if it would be useful when dealing with other transports e.g. grpc, databases, messaging.

However on second thoughts, those transports could have their own qualified 'protocol' tag.